src/KawaBot.Tests/BasicControllerTests.fs

```diff
@@ -37,3 +53,45 @@ type BasicControllerTests =
         let response = this._feeder.Send("/health") |> Async.RunSynchronously
         Assert.That(response.StatusCode, Is.EqualTo(HttpStatusCode.OK))
         Assert.That(response.Content.ReadAsStringAsync().Result, Is.EqualTo("OK"))
+
+    [<Test>]
+    member this.TestVerification() = 
+        let v = {
+            Token = "";
+            Challenge = "challenge";
+            Type = "";
+        }
+        let response = this._feeder.PostJSON("/slack", v) |> Async.RunSynchronously
```

Comment:
Lets use the Pascal case for abbreviations longer than 3 chars: `JSON` -> `Json`.